app/helpers/appropriate_body_helper.rb

```diff
@@ -41,13 +41,35 @@ def summary_card_for_teacher(teacher)
           {
             key: { text: "Status" },
             # FIXME: this is a placeholder as we cannot display a real status yet
-            value: { text: govuk_tag(text: "placeholder", colour: %w[grey green red purple orange yellow].sample) },
+            value: { text: govuk_tag(**induction_status_tag_kwargs(teacher)) },
           },
         ]
       )
     end
   end
 
+private
+
+  def induction_status_tag_kwargs(teacher)
+    case teacher.trs_induction_status
+    when 'RequiredToComplete'
+      { text: 'Required to complete', colour: 'grey' }
+    when 'Exempt'
+      { text: 'Exempt', colour: 'red' }
```

Comment:
```suggestion
    when 'Exempt'
      { text: 'Exempt', colour: 'orange' }
```
![Screenshot 2025-01-21 at 11 03 04](https://github.com/user-attachments/assets/868d9b00-d5bd-416a-ab01-d70646f33cb9)

Maybe orange? I feel like 'exempt' has a very different meaning to 'failed', so conflating them could be an issue.

Reply:
Pink! Purple!

Reply:
Yellow!